packages/suite/src/components/suite/PasswordStrengthIndicator/index.tsx

```diff
@@ -10,7 +10,7 @@ interface WrapperProps {
 const Wrapper = styled.div<WrapperProps>`
     display: flex;
     margin: 8px 0;
-    flex: 1;
+    /* flex: 1; */
```

Comment:
everything works nice even after commenting it out?

Reply:
huh, indeed that was probably not necessary. I noticed the strength indicator wasn't showing at all (latest develop), so I tried to figure out what happened to it. This is probably some left over from this.